### ISSUE TITLE
Feature : multilingual validation message centrally

### ISSUE THREAD

**Issue by rachanachag** [rxweb/rxweb#375]
There are 2 scenario :

1) I have used FormBuilder and Reactive form - in which validation is working properly for message "Maximum alloed length is {{requiredLength}}"  

2) I have used RxFormBuilder and model driven form - validation is not working properly 
In both cases I have used rx-translate 

please find stackblitz example [here](https://stackblitz.com/edit/rxweb-validation-message-with-parameters-6ycehw?file=src%2Fapp%2Fglobal%2Fglobal.component.html)

[referred document](https://docs.rxweb.io/ngx-translate-extension/validation-message)

**Comment by ajayojha** [comment]
@rachanachag right now this @rxweb/translate package feature is not supported to the @rxweb/reactive-form-validators package. I am considering this as a feature request.

**Comment by rachanachag** [comment]
Ok @ajayojha.

**Comment by rachanachag** [comment]
Please let us know if we are planning for support @rxweb/translate for @rxweb/reactive-dynamic-forms as well.

**Comment by ajayojha** [comment]
Please create a new ticket with appropriate use case of dynamic form.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by rachanachag** [comment]
ok

**Comment by ajayojha** [comment]
@rachanachag I have released the new version  2.1.0, please update the same. you have to add the below code in the root component:
```js
constructor(private rxTranslation: RxTranslation,private formBuilder: RxFormBuilder    ) {

ReactiveFormConfig.i18n = {
            validationMessage: () => {
                         return TranslationCore.getText("global.validationMessages")
            }
        };
        Object.defineProperty(ReactiveFormConfig.i18n, "language", {
            get: () => {
              
                return this.rxTranslation.language;
            }
        })


  }
```
Here is an [example](https://stackblitz.com/edit/rxweb-multilingual-error-messages?file=src/app/app.component.ts).
